CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog - FeedHenry Javascript SDK
 
+## 3.0.2 - 2018-04-30
+* Add missing argument to `sync.setQueryParams` definition
```

Comment:
The changelog and bumped version need to be in a separated commit. In this way, you must create as the follows example:
features: Add missing argument to `sync.setQueryParams` definition
features: Bumped Version


Examples of commits following this convention:

bug-fixes: git commit -a -m "fix(parsing): fixed a bug in our parser (RHMAP-1234)"
features: git commit -a -m "feat(parser): we now have a parser \o/ (FH-3456)"
breaking changes: